src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import cn from 'classnames';
+
+export const Navigation: React.FC = () => (
+  <nav
+    className="navbar is-light is-fixed-top is-mobile has-shadow"
+    data-cy="Nav"
+  >
+    <div className="container">
+      <div className="navbar-brand">
+        <NavLink
+          to="/"
+          className={({ isActive }) => cn(
+            'navbar-item',
+            { 'is-active': isActive },
+          )}
+        >
```

Comment:
Will you copy the same className if there will be 10 nav links?
Create a separate component for that.